rotkehlchen/chain/ethereum/uniswap/uniswap.py

```diff
@@ -81,8 +103,10 @@ def _get_balances_graph(
         graph_query: Callable,
     ) -> ProtocolBalance:
         """Get the addresses' pools data querying the Uniswap subgraph
+
+        Each liquidity position is converted into a <LiquidityPool>.
         """
-        address_balances: AddressBalances = {address: [] for address in addresses}
+        address_balances: DDAddressBalances = defaultdict(list)
```

Comment:
@LefterisJP @kelsos here the fix for returning only the addresses that have balances.

Reply:
thanks